modules/dists/CyclicDist.chpl

```diff
@@ -466,12 +457,13 @@ class LocCyclic {
 
   const myChunk: domain(rank, idxType, true);
 
-  proc init(param rank, type idxType, locid, dist: Cyclic(rank, idxType)) {
+  proc init(param rank, type idxType, locid,
+            distStartIdx: rank*idxType, distLocDims) {
```

Comment:
Another instance of passing the components of ``dist``. Can you include your reasoning in the PR message? If it's not necessary, I'd just as soon leave it as it was.